### PR TITLE
Improve stableness

### PR DESCRIPTION
**Pull request by MV-GH**
Improve stableness of many composables. The root cause is navHostController. It is considered unstable

 "skippableComposables": 549, -> 582
 "knownUnstableArguments": 393, ->234
examples of before

```
restartable scheme("[androidx.compose.ui.UiComposable]") fun PostEditActivity(
  stable postView: PostView
  unstable accountViewModel: AccountViewModel
  unstable navController: NavController
)

restartable scheme("[androidx.compose.ui.UiComposable]") fun CommunityListActivity(
  unstable navController: NavController
  unstable accountViewModel: AccountViewModel
  stable selectMode: Boolean = @static false
  stable siteViewModel: SiteViewModel
  stable blurNSFW: Boolean
  stable drawerState: DrawerState
)

```
after

```
restartable skippable scheme("[androidx.compose.ui.UiComposable]") fun PostEditActivity(
  stable postView: PostView
  stable accountViewModel: AccountViewModel
  stable appState: JerboaAppState
)

restartable skippable scheme("[androidx.compose.ui.UiComposable]") fun CommunityListActivity(
  stable appState: JerboaAppState
  stable accountViewModel: AccountViewModel
  stable selectMode: Boolean = @static false
  stable siteViewModel: SiteViewModel
  stable blurNSFW: Boolean
  stable drawerState: DrawerState
)


```

Metrics files
[compose_metrics_old.zip](https://github.com/dessalines/jerboa/files/12059804/compose_metrics_old.zip)
[compose_metrics.zip](https://github.com/dessalines/jerboa/files/12059805/compose_metrics.zip)

Also redid the baseline profiles

I have still many ideas, but I ll keep those for other PRs. This one is big enough already.

### CODE REVIEW

**Review by dessalines — APPROVED** [review]
Tested, and this works well. Thx especially for moving the navcontroller out of the composables.

I'll let @twizmwazin look this over before merging.

**Review by twizmwazin — APPROVED** [review]

**Pull request merged**
